samples/dnn/human_pose.cpp

```diff
@@ -95,18 +99,22 @@ int main(int argc, char **argv)
     }
 
     // read the network model
-    Net net = readNet(modelBin, modelTxt);
+    Net net = readNetFromONNX(model);
     // and the image
     Mat img = imread(imageFile);
     if (img.empty())
     {
         std::cerr << "Can't read image from the file: " << imageFile << std::endl;
         exit(-1);
     }
-
     // send it through the network
-    Mat inputBlob = blobFromImage(img, scale, Size(W_in, H_in), Scalar(0, 0, 0), false, false);
-    net.setInput(inputBlob);
+    Mat blob = blobFromImage(img, scale, Size(W_in, H_in), Scalar(0, 0, 0), false, false);
+
+    std::vector<int> order = {0, 2, 3, 1};
+    Mat outBlob; // inp: NCHW, out: NHWC
+    transposeND(blob, order, outBlob);
```

Comment:
There is blobFromImageWithParams: https://docs.opencv.org/5.x/d6/d0f/group__dnn.html#gadc12e5f4a801fd3c1d802f4c8c5d311c. It handles NCHW -> NHWC conversion too.

Reply:
fixed @asmorkalov